src/zarr/core/config.py

```diff
@@ -106,8 +108,8 @@ def enable_gpu(self) -> ConfigSet:
                     ],
                 },
             },
-            "async": {"concurrency": 10, "timeout": None},
-            "threading": {"max_workers": None},
+            "async": {"concurrency": 1 if IS_WASM else 10, "timeout": None},
+            "threading": {"max_workers": 1 if IS_WASM else None},
```

Comment:
The idea here is to block spawning any new threads. I'm not sure if `concurrency` should be set to `1` or if we can safely use `10`.